config/default.yml

```
@@ -282,8 +282,15 @@ Rails/FindEach:
   StyleGuide: 'https://rails.rubystyle.guide#find-each'
   Enabled: true
   VersionAdded: '0.30'
+  VersionChanged: '2.8.2'
   Include:
     - app/models/**/*.rb
+  IgnoredMethods:
+    # Methods that don't work well with find_each
```

This is a nitpick :-)

```suggestion
    # Methods that don't work well with `find_each`.
```